packages/flutter/test/widgets/column_test.dart

```diff
@@ -777,4 +777,57 @@ void main() {
     expect(renderBox.size.width, equals(0.0));
     expect(renderBox.size.height, equals(100.0));
   });
+
+  testWidgets('Default Column.spacing value', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: Column(),
+      ),
+    );
+
+    expect(tester.widget<Column>(find.byType(Column)).spacing, 0.0);
+  });
+
+  testWidgets('Can update Column.spacing value', (WidgetTester tester) async {
+    Widget buildRow({ required double spacing }) {
```

Comment:
This should probably be called `buildColumn` ?

Reply:
Good catch, i missed it when copying from column class.